app/javascript/components/item-view.vue

```diff
@@ -0,0 +1,69 @@
+<script setup lang="ts">
+import type { Item } from '../api/items';
+import { computed, ref } from 'vue';
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>  'computed' is defined but never used                  @typescript-eslint/no-unused-vars